defaults/config.js

```diff
@@ -86,6 +86,28 @@ module.exports = {
 	//
 	prefetchStorage: false,
 
+	//
+	// Allow uploading files to the server hosting TheLounge.
+	// If enabled, The Lounge will store files in the `${THELOUNGE_HOME}/storage` folder.
+	//
+	// Images will not expire or be removed by TheLounge, so be wary of memory and your hardware.
+	//
+	// @type     boolean
+	// @default  false
```

Comment:
wrong default.

Reply:
No -- wrong value imo. I think the default should be false, as this is unwise on public instances at the moment.

Reply:
Agreed :)

Reply:
fixed

Reply:
I think it's fine to start with until it gets stable, but I think in the long run we should set this to `true` by default:

- Config sets instance in private mode by default
- For discoverability, I think we should have features enabled by default with a way to disable if necessary, but not the other way around (I have the same opinion about prefetch)

Reply:
I mean, discoverability will be better when we get #1950 in and we can add the server settings to the frontend for admins.